privacy_evaluator/output/user_output.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+import pandas as pd
+from matplotlib import pyplot as plt
+
+from typing import Tuple
+
+
+class UserOutput:
```

Comment:
please document the parameters of this class (line 14-16)

Reply:
done